app/controllers/api/v1/meetings_controller.rb

```diff
@@ -90,7 +90,8 @@ def authorized_as_viewer?(access_code:)
       def authorized_as_moderator?(access_code:)
         (params[:access_code].present? && access_code == params[:access_code]) ||
           @room.anyone_joins_as_moderator? ||
-          current_user&.shared_rooms&.include?(@room)
+          current_user&.shared_rooms&.include?(@room) ||
+          current_user&.rooms&.include?(@room)
```

Comment:
You can just do `@room.user_id == current_user.id` to avoid doing another query

Reply:
right thanks